bin/zopen-init

```diff
@@ -222,10 +214,63 @@ init()
       fi
     fi
   fi
+}
+
+determineStatsCollection()
+{
+  if isIBMHostname; then
+    isHostIBM=true
+  fi
+  if [ -n "$enableStats" ] && $enableStats; then
+    isCollectingStats=true
+  elif [ -n "$enableStats" ] && ! $enableStats; then
+    isCollectingStats=false
+  elif $isHostIBM; then
+    printAttention "Attention: Turning on usage statistics as your host is an IBM host. If this is incorrect, rerun with --noenable-stats"
+    isCollectingStats=true
+  else
+    printInfo "Do you want to contribute anonymous usage statistics to help improve z/OS Open Tools? (press y to collect stats or n for no): "
+    read collectStats < /dev/tty
+    if [ "y" = "${collectStats}" ]; then
+      isCollectingStats=true
+    fi
+  fi
+
+  # Undocumented envar, for CI/CD tracking
+  if [ -n "${ZOPEN_IS_BOT}" ]; then
+    isBot=true
+  fi
+}
+
+init()
+{
+  if ${reinitExisting}; then
+    printHeader "Re-initializing zopen framework"
+  elif ${refresh}; then
+    printHeader "Refreshing zopen framework"
+  else
+    printHeader "Initializing zopen framework"
+  fi
+
+
+  printDebug "Validating input parameters"
+  case "${fslayout}" in
+    "usrlclz") zopen_pkginstall="usr/local/zopen"    ;;
```

Comment:
i thought we were going to get rid of these choices? although that could be a separate PR

Reply:
Yeah, I think we should handle this in another PR, I just moved the code around in this change.